badCode.rb

```diff
@@ -0,0 +1,5 @@
+def badName
+  if something
+    test
+    end
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Layout/EndAlignment: end at 4, 4 is not aligned with if at 2, 2.